sdk/translation/azure-ai-translation-document/src/test/java/com/azure/ai/translation/document/DocumentTranslationClientTestBase.java

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.translation.document;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.test.TestProxyTestBase;
+import com.azure.core.test.models.TestProxySanitizer;
+import com.azure.core.test.models.TestProxySanitizerType;
+import com.azure.core.util.Configuration;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobClientBuilder;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobContainerClientBuilder;
+import com.azure.storage.blob.sas.BlobContainerSasPermission;
+import com.azure.storage.blob.sas.BlobServiceSasSignatureValues;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class DocumentTranslationClientTestBase extends TestProxyTestBase {
+
+    private static final String HOST_NAME_REGEX = "(?<=http://|https://)(?<host>[^/?\\\\.]+)";
+    private static final String REDACTED = "REDACTED";    
+
+    @Override
+    public void beforeTest() {
+        super.beforeTest();
+    }
+
+    DocumentTranslationClient getDocumentTranslationClient() {
+        return getDTClient(getEndpoint(), getKey());
+    }
+
+    DocumentTranslationClient getDTClient(String endpoint, String key) {
+        DocumentTranslationClientBuilder documentTranslationClientbuilder = new DocumentTranslationClientBuilder()
+                .endpoint(endpoint)
+                .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC));
+
+        if (interceptorManager.isPlaybackMode()) {
+            documentTranslationClientbuilder.httpClient(interceptorManager.getPlaybackClient());
+        } else if (!interceptorManager.isLiveMode()) {
```

Comment:
Remove this else if and move it to L60.
Since this is an else-if for record scenarios for the `DocumentTranslationClient` we aren't adding the recording policy and hence no requests are getting recorded.

Reply:
Moved